tests/baselines/reference/noImplicitOverride.errors.txt

```diff
@@ -0,0 +1,86 @@
+tests/cases/compiler/noImplicitOverride.ts(11,5): error TS90034: Class member 'toString' must be marked 'override' when noImplicitOverride is enabled (augmented from Object.Object)

```

Comment:
Something's wrong with the way this is being rendered. We shouldn't have `Object.` in front of all of these. (Except e.g. `Object.toString` which is actually a property of Object.)

Reply:
Fixed